### DISCUSSION TITLE
Horse como cliente WebSocket?

### DISCUSSION THREAD

**Discussion by ezlage** [HashLoad/horse#193] [Q&A] [answered]
Olá! 

Acabei de ser apresentado ao Horse. Gostaria de saber se há uma página com documentação, manuais, tutoriais, etc. Caso não, poderiam me dizer se é possível usar o Horse como cliente WebSocket? Saberiam de alguma implementação ou código de exemplo?

Obrigado e parabéns! O Horse é simples e muito útil!

**Comment by viniciussanchez** [comment, marked as answer]
Bom dia, desculpa a demora... mas respondendo sua pergunta, não costumo usar o Horse como WebSocket não. Existem projetos específicos pra isso e muito bons no GitHub como por exemplo o BirdSocket (https://github.com/mateusvicente100/bird-socket-server)
Com relação a documentação do Horse, o que temos é o que está no Github mesmo, e artigos e vídeos do YT.
Temos um canal no Telegram muito ativo tambem.
https://www.youtube.com/c/HashLoad
https://www.youtube.com/channel/UChvY9l3AR7igAKX1TmdpAKw
https://www.youtube.com/channel/UCN8rDx_CAXUJk2IcYRm4aPQ
https://www.youtube.com/c/AdrianoSantosTreina
https://www.youtube.com/c/AlexandreMagnoSouza
Entre outros...
